TestingApp/Podfile

```diff
@@ -0,0 +1,6 @@
+platform :osx, '10.12'
```

Comment:
Remove remove it since we don't use it now

Reply:
done